pom.xml

```diff
@@ -1456,7 +1456,7 @@
       <dependency>
         <groupId>org.mycore.sword</groupId>
         <artifactId>easy-sword2-lib</artifactId>
-        <version>1.0.4</version>
+        <version>1.0.5-SNAPSHOT</version>
```

Comment:
I would use a released version instead of "*-SNAPSHOT".

Reply:
will be released in 2020-02